radixdlt-core/radixdlt/src/main/java/com/radixdlt/environment/rx/RxEnvironmentModule.java

```diff
@@ -72,12 +73,22 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
 /**
  * Environment utilizing RxJava
  */
-public class RxEnvironmentModule extends AbstractModule {
+public final class RxEnvironmentModule extends AbstractModule {
+
+	public static final class Runners {
+		public static final String SYNC = "sync";
+		public static final String MEMPOOL = "mempool";
+		public static final String APPLICATION = "application";
+		public static final String CHAOS = "chaos";
+		public static final String CONSENSUS = "consensus";
```

Comment:
Better if these were enums and outside of RxEnvironment.

Reply:
Concept of "ModuleRunner" should extend to Deterministic tests at some point.

Reply:
can't use enums because we need a constant String for ProvidesIntoMap annotation